### PR TITLE
Refactored functions in favoritesQuery.js

### PR DESCRIPTION
**Pull request by ShawnToubeau**
Combined `createLiveLinkTable` and `createUserFavoriteTable` into one function `createTable`.

### CODE REVIEW

File: resources/scripts/explore.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ var goToLiveLink = function (liveLinkId) {
     window.open('/livelink/public/' + currentRecent.en_url, '_blank');
 }
 
-var createLiveLinkDataTable = function (obj) {
+var createLiveLinkDataTable = function (obj) {
```
Comment (Ethan-Arrowood) [inline resources/scripts/explore.js]: Use `const` here

File: utility/favoritesQuery.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ var setFavorites = function(req, res, favoriteObj) {
   });
 };
 
-var setLiveLink = function(req, en_url, id) {
+var setLiveLink = function(req, res, en_url, id) {
```
Comment (Ethan-Arrowood) [inline utility/favoritesQuery.js]: update all `var` to appropriate `let` and `const`

**Comment by ShawnToubeau** [comment]
Made the changes! Also tried your suggestion for the pack-lock.json but that didn't seem to work

**Comment by Ethan-Arrowood** [comment]
These changes have merge conflicts with #32 
Instead of merge this PR I'm going to show you how you can make changes to my PR branch and we can submit the PR as co-authors.

1. Some assumptions:
  - Your `upstream` branch is HarperDB
  - Your `origin` branch is your fork
2. Clear out your deps:
  - run `rm -rf node_modules/` and `rm -f package-lock.json` from the `HarperDB_Studio` directory.
3. Run `git remote add ethan-arrowood https://github.com/Ethan-Arrowood/HarperDB_Studio.git`
4. Run `git fetch ethan-arrowood add-hdb-connect`
5. Run `git checkout ethan-arrowood/add-hdb-connect`
6. Make your edits and when you're ready to commit and push your changes do the following:
  - `git add .`
  - `git commit -m '<insert descriptive commit message>'`
  - `git push origin add-hdb-connect` (Remember origin is your fork)
7. Navigate to GitHub and go to *your* fork.
8. Open a PR from your fork's branch `add-hdb-connect` to my fork branch `add-hdb-connect`

If there are any issues you know how to contact me :)

**Pull request closed without merging**